### ISSUE TITLE
errors - fixed  (maya2022.3)

### ISSUE THREAD

**Issue by sitorender** [mhdmhd/MayaSceneConverter#4]
Hi, it was not working for me (Maya2022.3 - Win)

I toke 2 ERRORS in ConverterUI.py:

line 21: reload(Conv)
corrected with:
from importlib import reload
reload(Conv)

line 677: ptr = wrapInstance(long(qtCtrl), wrap)
corrected switching long with int:     ptr = wrapInstance(int(qtCtrl), wrap)

Just googling these "errors" (not really errors, just "reload" and "long" no longer supported in phyton3, right?) I found a simple solution.

Anyway THANK YOU VERY VERY MUCH for the script, really very useful.

---------------------
Update: I don't know what I'm doing wrong, trying to change standard phong shader to Renderman PxrSurface... new Pxrsurface is created but original selected mesh has no shader assigned, and if phong had linked textures PxrSurface has no linked textures. I'm looking to the code and script editor to find a solution but I'm not a programmer and I'm unable to do it.
Any tips? Thanks

**Comment by mhdmhd** [comment]
Hi, I am very sorry, I already fixed that but forgot to update the release section.
you could get it from the Code section for now.

For the other problem, If I understand correctly you have created a new rule for Renderman but it doesn't behave as expected.
I can't really help you unless you explain a lot more or at least send me the rule file, also I don't have Renderman at the moment, sorry.

**Comment by sitorender** [comment]
IT WORKS, it works........ **thanks a lot, it's amazing!**

Totally my fault, I didn't **select first** the name of the shaders, like "**phong**"-"**PxrSurface**"- **->**
I just selected only their attributes, like "color"-"diffuseColor"- ->       😅 I think that was the problem.

I had only a small issue about the name of the new rule: in the SaveFile window i left the name _"mayaSoftware_To_renderman.json"_, in the rules folder the name is that, but in the drop-down menu it appears like _"mayaSoftware_To_renderma"_. If i try to convert it doesn't work and script editor says there's no file named mayaSoftware_To_renderma.
If I rename the file _"mayaSoftware_To_renderman.json"_ to _"mayaSoftware_To_renderma.json"_ it works.

If I try to save the rule with shorter name, like only  _"renderman"_,  in the drop-down menu I have _"renderma"_.

Anyway, renaming the file in the rule folder to match the one in the drop-down menu works, so not really a problem.

**Thank you a lot**, this script can let me save hours of work!
I tried to modify a simple script yesterday and i spent all the night on that, it works but your one is so amazing and much more complex, I guess you put a big effort to do that and I'm really grateful to you for sharing it for free.
